Simplified/NYPLMyBooksDownloadCenter.m

```diff
@@ -444,6 +444,7 @@ - (void)deleteLocalContentForBookIdentifier:(NSString *const)identifier account:
            error:&error]){
         NYPLLOG_F(@"Failed to remove local content for download: %@", error.localizedDescription);
       }
+      break;
```

Comment:
I added an extra break here because it didn't make sense to me that fallthrough should be allowed. Hopefully this can be sanity-checked.